qqqwww123cdv/hometask-4/my-hash/my_hash.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# realisation of Hash
+class MyHash
+  attr_reader :myhash
+
+  include Enumerable
+
+  def initialize
+    @hash = []
+  end
+
+  def [](key)
+    myhash[key]
+  end
+
+  def []=(key, value)
+    @hash.push([key, value])
+  end
+
+  def clear
+    @hash = reject { |value| value }
```

Comment:
we need just to pass a truthy value in `reject`, so using the argument is redundant here